dac-ansible/README.md

```diff
@@ -2,13 +2,17 @@
 
 Install data-acc demo with ansible. It creates a bunch of OpenStack VMs, then uses ansible to install a development data-acc enviroment.
 
-To run this set of playbooks, first we create some VMs in OpenStack:
+To run this set of playbooks, first we create some VMs in OpenStack.
+
+Source your OpenStack RC, eg:
 
     . openrc
+
+Edit create-servers.py to set your NETWORK_NAME and KEYPAIR_NAME and then run:
```

Comment:
I would rather the script took this from some environment variables, but that can be a follow up change. I am suprized just changing those two things (and dropping the gateway) was enough to make it work, but cool if that is the case :)